.github/CODEOWNERS

```diff
@@ -55,7 +55,6 @@
 /src/lowertest                      @MaterializeInc/compute
 /src/lowertest-derive               @MaterializeInc/compute
 /src/metabase                       @benesch
-/src/mz                             @MaterializeInc/devex
```

Comment:
Can we transition this to @MaterializeInc/adapter?

Reply:
Assumed this would fall under `/adapter` or `/cloud`, but wanted to double-check ownership internally before committing a team. @chaas

Reply:
Possibly it should be a new @.MaterializeInc/integrations team, or the wider umbrella of @.MaterializeInc/surfaces, rather than @.MaterializeInc/adapter, but I do think this wants to live under the surfaces/integrations team—vs. the src/cloud-api package, which could reasonably live under @.MaterializeInc/cloud.

Reply:
I like the idea of adding a new @.MaterializeInc/integrations GH team, since that reflects the true ownership. Also fine with @MaterializeInc/surfaces in the meantime.

Reply:
Ok, added a new @MaterializeInc/integrations team. @bobbyiliev is the only member. Are there other folks we should add? If so, since I'll be OOO tomorrow through Friday, just ask in #operations for someone to add members to the "engineering.team.integrations" Okta group.